Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -127,9 +127,13 @@ family of functions. When writing tests using the testing library, implement
   }
 }
 
-The use of `async` and `throws` is optional. If teardown is needed, declare your
-test suite as a `final` class or as an actor rather than as a structure and
-implement `deinit`:
+In XCTest, you can also schedule work to run after a test using the [`tearDown()`](https://developer.apple.com/documentation/xctest/xctest/3856482-teardown)
+family of functions. When writing tests using the testing library, adopt structured
+  [concurrency](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/concurrency/) 
+and take advantage of Swift to handle cleanup for you where possible rather 
+than relying on a dedicated teardown method. If teardown is necessary, declare
+your test suite as a `final` class or actor and implement `deinit`. Your cleanup
+operations should be quick, synchronous, and non-throwing.
```

Comment:
Not sure we need to say that cleanup operations should be "quick"; it's perfectly valid to write a long-running cleanup task if you want. Of course, overall test duration will still be subject to any time limit trait which has been applied, if any.